src/slices/filters.ts

```diff
@@ -63,15 +64,15 @@ const reducers = {
     if (!state.static) {
       state.static = [];
     }
-    const { filter: targetFilter, shouldSelect } = payload;
-    const matchingFilter = state.static.find(storedSelectableFilter => {
-      const { selected:_, ...storedFilter } = storedSelectableFilter;
+    const { filter: targetFilter, shouldSelect, displayName } = payload;
+    const matchingFilter = state.static.find(storedDisplayableFilter => {
+      const { selected:_, displayName:__, ...storedFilter } = storedDisplayableFilter;
```

Comment:
if we don't remove the unused properties (`selected` and `displayName`) does typescript complain? was thinking it might be slightly nicer to just delete line 69

Reply:
It doesn't complain, will update!